jshift-kit-spring-boot/src/main/java/io/jshift/springboot/generator/SpringBootGenerator.java

```diff
@@ -32,22 +48,6 @@
 import java.util.zip.ZipInputStream;
 import java.util.zip.ZipOutputStream;
 
```

Comment:
What does this change do?

Reply:
Apparently nothing much. It was intellij

Reply:
Fixed